### DISCUSSION TITLE
Preview avatar issue , no thumbnail preview

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3439] [Hubs Cloud Questions] [answered]
After updating my stack in AWS , I have an issue with the avatars preview screen . I can see the avatar anymore . 

Have you already seen this issue ? and how to resolve it 
Thx 
![Capture_decran_2020-11-27_08 45 07](https://user-images.githubusercontent.com/58043917/100451167-39b14780-30b7-11eb-886b-84fd867013aa.png)

**Comment by brianpeiris** [comment]
Can you describe what updates you did to your stack?
What version of the stack are you running, personal or enterprise?
If it's an enterprise stack, how many servers are running?
If you open your browser's developer tools, view the "Network" tab, what status codes do you see for those image requests?

**Reply by eyesnareinc** [reply]
Same thing is going on with my stack. 
![avatarBrowseThumbnail (1)](https://user-images.githubusercontent.com/66790476/100527547-6d689c00-31a1-11eb-987f-14a0a6c07ed5.jpg)

- Template has been updated to the latest version.
- Hubs Cloud Personal version on AWS 
- Selected Avatar is showing but the avatar thumbnails are not showing
- No status codes are showing up on the console (Firefox) (200)
- Spoke asset thumbnails are also not showing up

![avatarBrowseThumbnail (1)](https://user-images.githubusercontent.com/66790476/100527808-00a2d100-31a4-11eb-95be-5ba59faec9db.jpg)

![Screenshot 2020-11-28 180213](https://user-images.githubusercontent.com/66790476/100527809-04ceee80-31a4-11eb-88fc-08ba5f24e818.png)

**Comment by eyesnareinc** [comment, marked as answer]
Did you find a solution for this? I have not. :)

**Reply by robinkwilson** [reply]
We need a screenshot of your Browser Developer tools > Network tab -- see if any requests are failing.
And/or can you "Inspect" an image element using your Browser's developer tools so we know what image it's trying to fetch?

**Reply by robinkwilson** [reply]
For "Network tab" example:
![image](https://user-images.githubusercontent.com/29560735/103030124-045e1500-4510-11eb-82aa-be08351f891d.png)

For "Inspect" example: (Developer tools > Upper left is the cursor + box > click to highlight in blue and inspect HTML elements on the current page)
![image](https://user-images.githubusercontent.com/29560735/103030200-2192e380-4510-11eb-97ac-c773d1873537.png)

**Reply by eyesnareinc** [reply]
Thank you! I rebuilt the stack to resolve the problem. I wasn't getting request errors from the Network tab.

**Reply by robinkwilson** [reply]
Woof, what a hassle, glad you're good to go! Though I'm still puzzled by what went wrong!
